.pack.mk

```diff
@@ -22,17 +22,25 @@ ifeq (${VERSION}, $(filter ${VERSION}, 1.10))
 endif
 
 prepare:
-	rm -rf build packpack
-	git clone https://github.com/packpack/packpack.git
+#	rm -rf build packpack
+#	git clone https://github.com/packpack/packpack.git
 
 package: prepare
 	if [ "${TARANTOOL_SERIES}" != "1.10" ]; then \
 		if [ -n "${GIT_TAG}" ]; then \
 			export VERSION="$$(echo ${GIT_TAG} | sed 's/-/~/')"; \
 		else \
-			export VERSION="$$(echo ${GIT_DESCRIBE} | sed ${SED_REPLACE_VERSION_REGEX} | sed 's/-/~/').dev"; \
+			if [ "${OS}" = "alpine" ]; then \
+				if git describe --abbrev=0 | grep -q "entrypoint" ; then \
```

Comment:
Looks like `$$(echo ${GIT_DESCRIBE} | grep -q "entrypoint")` should work fine.

Reply:
I agree that "--abbrev=0" seems excess, and I remove it. But construction with "$$(echo "  seems excessive too.

Reply:
Looks like fixed.